metadata-ingestion/src/datahub/ingestion/source/redash.py

```diff
@@ -529,6 +536,16 @@ def _emit_dashboard_mces(self) -> Iterable[MetadataWorkUnit]:
                 # Continue producing MCE
                 dashboard_slug = dashboard_response["slug"]
                 dashboard_data = self.client.dashboard(dashboard_slug)
+
+                # Check dashboard queries datasource
+                for widget in dashboard_data["widgets"]:
```

Comment:
Will widgets field always be populated?

Reply:
Yes! The widgets field is set to a empty list if there's no widget on dashboard_data